examples/6_show_image_transforms.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
+
+from torchvision.transforms import ToPILImage, v2
+
+from lerobot.common.datasets.lerobot_dataset import LeRobotDataset
```

Comment:
@marinabar could we add a documentation as header that looks like previous examples?

Reply:
Yes I renamed the files